frontend/public/app.js

```diff
@@ -80,4 +87,46 @@ class Application {
     renderWorkers() {
         console.log("Rendering workers");
     }
+
+    showNotificationByResponseStatus(responseStatus) {
+        this.setCloseAlertOnClick();
+
+        if (responseStatus>= 200 && responseStatus < 300) {
+            this.setAlertMessageContent("Fruit succesfully created", "fa fa-check-circle", "pf-c-alert pf-m-success pf-m-inline");
+        } else {
+            this.setAlertMessageContent("An unexpected error occurred. Please try again later", "fa fa-exclamation-circle", "pf-c-alert pf-m-danger pf-m-inline");
+        }
+
+        this.showAlertForDuration(3000);
+    }
+
+    setCloseAlertOnClick() {
+        const closeAlertButton = document.getElementById("closeAlertButton");
+        closeAlertButton.addEventListener("click", (event) => {
+            this.hideElement(alertMessage);
+        });
+    }
+
+    setAlertMessageContent(title, iconClass, alertTypeClass) {
+        const alertTitle = document.getElementById("alertTitle");
+        const alertIcon = document.getElementById("alertIcon");
+        const alertMessage = document.getElementById("alertMessage");
+
+        alertTitle.innerHTML = title;
+        alertIcon.className =  iconClass;
+        alertMessage.className = alertTypeClass;
+    }
+
+    hideElement(element) {
+        element.style.visibility = "hidden";
+    }
+
+    showAlertForDuration(duration) {
+        const alertMessage = document.getElementById("alertMessage");
+        alertMessage.removeAttribute("style"); 
+        clearTimeout(this.alertTimeoutReference);
+        this.alertTimeoutReference = setTimeout(() => {
```

Comment:
Why do we store this `alertTimeoutReference` var in an external scope?

Reply:
Reasoning for this was that I needed some reference to the timeout, so each time an alert is called, it always last for the set duration and why the clearTimeout is there. 

Otherwise say a error notification is shown, where 2 seconds has passed on a 3 second duration. Then a success notification is shown after another attempt. This notification would be closed after 1 second instead of 3, due to the previous timeout, which was why I put it in an external scope. 

Unless this can be done differently ? :thinking: